python/pylibcugraph/setup.py

```diff
@@ -12,39 +12,41 @@
 # limitations under the License.
 
 import os
-import sys
-import sysconfig
 import shutil
 
-from setuptools import setup, find_packages, Command
-from setuptools.extension import Extension
-from setuputils import get_environment_option
+from setuptools import find_packages, Command
+from skbuild import setup
+# FIXME: Not necessary
```

Comment:
Same request as with the other setup.py, please address as many FIXMEs as you can.

Reply:
Addressed